### ISSUE TITLE
Add support for accessibleSlider/NumberPicker to have verbose value text on focus

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#482]
In GFLB and regular I am finding that it is a bit of a pain to support cases where we want a basic value text to be read out each time the value changes, and a more verbose one to be read out only on focus.

I am finding myself implementing this manually in GFL/MassControl and GFLB/GFLBMassControl.

**Comment by terracoda** [comment]
@zepumph, having **an option** to provide the extra dynamic state information on focus through `aria-valuetext` would be great.

It is not always needed. For example, in JT, RIAW and OL we didn't need this option.

**Comment by zepumph** [comment]
Indeed! I agree that an option would be the best forward. I talked with @mbarlow12 about this, since he has worked a lot in AccessibleNumberSpinner and AccessibleSlider regarding aria-valuetext, and we thought through a few ways to improve their apis.

* It seems like Accessibility.js should have aria value text getters and setters.
* It seems like the two accessible traits should have the same api for aria value text. Right now there are different option names for both, and they aren't implemeted the same way, which is how we got to this issue from https://github.com/phetsims/gravity-force-lab-basics/issues/96 because my work around weirdness in `MassControl.js` (REGULAR) didn't apply easily to `GLFBMassControl.js`.
* Perhaps we create options (or one single option callback) to handle the proper hooks of changing the text (like on focus/on property change/etc). But instead perhaps we leave most of it up to the client for specific use cases. 
* Though, as a note about the bullet directly above, it seems strange to have accessible trait Types responsible for SOME of the aria-valuetext setting, and then have the client usage responsible for the rest. In the worst case that could lead to race conditions and in the best case it involves the necessity of fully understanding the underlying implementation before adding a layer of listener to change value text on top of the Slider/NumberPicker.

I'm going to try to think through a prototype of way to consolidate apis, and then I will look for a way to best add in GFL needs to it.

**Comment by zepumph** [comment]
After the above thoughts. I think the best way forward is to use this issue to redesign the ariaValueText api for a11y. Renaming the issue.
